tools/ts-generator/types/types_gen_js.py

```diff
@@ -378,6 +394,19 @@
 """
 
 
+def get_subtype(type):
```

Comment:
what is subtype? Maybe we should rename these methods. For example for this type: `Optional<T>` we would call `T` the `value_type` and `Optional<T>` is just the type. I'm assuming by subtype you mean value_type? If so can we rename that? I think it would be more clear

Reply:
Sure, I'll change to `get_value_type`